test/ruby_ui/masked_input_test.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class RubyUI::MaskedInputTest < Minitest::Test
+  include Phlex::Testing::ViewHelper
+
+  def test_render
+    output = phlex_context do
+      RubyUI.MaskedInput(data: {maska: "#####-###"})
```

Comment:
I was wondering if we should try to make it more generic like using mask as the data attribute name. But doing so, we will have to map all data attributes expected by maska. It will be simpler if we explicitly tell the users that maska is doing the mask job and point them to the mask docs.

Reply:
Yeah, I think we are tied to the js lib API anyway, it would be difficult to have a generic approach here since each mask lib have distinct config formats.

Reply:
I agree with @cirdes.  I think `mask` is a _much_ better name for the attribute.

Reply:
I agree 100%. This seems odd at first but it makes it explicitly clear that something is going on and if they look into it it will show we are using maska which will make sense.